cg/server/app.py

```diff
@@ -128,6 +128,9 @@ def _register_admin_views():
     ext.admin.add_view(admin.FlowcellView(Flowcell, ext.db.session))
     ext.admin.add_view(admin.AnalysisView(Analysis, ext.db.session))
     ext.admin.add_view(admin.InvoiceView(Invoice, ext.db.session))
+    ext.admin.add_view(
+        admin.IlluminaFlowCellView(IlluminaSequencingRun, ext.db.session, name="Illumina Flow Cell")
+    )
```

Comment:
I am confused... the FlowCell is the RunDevice now, is it not?

Reply:
resolved in person